images/openshift-enterprise-hyperkube.yml

```diff
@@ -1,11 +1,10 @@
 container_yaml:
   go:
     modules:
-    - module: github.com/openshift/origin
+    - module: github.com/openshift/kubernetes
 content:
   source:
-    alias: ose
```

Comment:
What is this line intended to do? I am only sure that ose should no longer be referenced since it is a private fork derived from the origin repo.

Reply:
Answered on slack, updated to use an explict git reference instead.